packages/apidom-reference/test/dereference/strategies/asyncapi-3/security-scheme-object/fixtures/operation-object/root.json

```
@@ -0,0 +1,16 @@
+{
+  "asyncapi": "3.0.0",
+  "components": {
+    "operations": {
+      "operation1": {
+        "security": {
```

Should be an array of Security Schemes
```suggestion
        "security": [{
```